fmn/rules/pagure.py

```diff
@@ -41,6 +41,33 @@ def pagure_specific_project_filter(config, message, project=None, *args, **kw):
     return valid
 
 
+@hint(categories=['pagure'], invertible=False)
+def pagure_specific_project_tag_filter(config, message, tags=None, *args, **kw):
+    """ Particular pagure tag projects
```

Comment:
Maybe change this first line of the docstring to "Particular pagure project tags"